eip3198/tests.go

```diff
@@ -151,6 +151,9 @@ func testBaseFeeGlobalPreHertz(boundContract *bind.BoundContract) error {
 	var result *big.Int
 	err = boundContract.Call(nil, &[]interface{}{&result}, "basefee_global")
 	expectedErrorMsg := "invalid opcode: BASEFEE"
+	if err == nil {
+		return fmt.Errorf("Expected %s, got <nil>", expectedErrorMsg)
```

Comment:
maybe replace `got <nil>` to `got 'no error' ` ?

Reply:
OK